src/internal/packager/sbom/viewer.go

```diff
@@ -44,8 +47,8 @@ func (b *Builder) createSBOMHTML(filename string, goTemplate string, jsonData []
 	}{
 		ThemeCSS:  b.loadFileCSS("theme.css"),
 		ViewerCSS: b.loadFileCSS("styles.css"),
-		List:      template.JS(b.jsonList),
-		Data:      template.JS(jsonData),
+		List:      template.JS(safeJSONList),
+		Data:      template.JS(safeJSONData),
```

Comment:
Should have checked earlier, but this actually breaks the way we are including these values into the sbom html files:

![image](https://github.com/defenseunicorns/zarf/assets/3977569/186f51fa-6a2d-43d9-8c76-bd858b41a864)

(note the escaped values)

![image](https://github.com/defenseunicorns/zarf/assets/3977569/9a3cdeab-66d4-4665-ae0e-62550b535ab3)

simple repro:

```
zarf package create examples/dos-games/
zarf package inspect zarf-package-dos-games-amd64-1.0.0.tar.zst -s
```